database/db.js

```diff
@@ -2,16 +2,60 @@ var mongoose = require('mongoose');
 mongoose.connect(`mongodb://localhost/ARList`, {useNewUrlParser:true});
 // create schema
 const AReviewSchema = new mongoose.Schema({
-  reviewId: { type: Number, unique: true },
+  reviewId: Number,
   reviewRating: Number,
-  reviewDate: Date(),
+  reviewDate: String,
   reviewText: String,
   reviewerName: String
 })
 
 // create model
-const AReview = mongoose.model('Review', AReviewSchema);
+var AReview = mongoose.model('review', AReviewSchema);
 
+// random name generator
+var RandomNameGenerator = () => {
+  let firstName = ["Julia", "Rob", "Jenny", "Kai", "Jon"],
+    lastName = ["Gens", "Nolan", "Shamoo", "Dong", "Yang"]
```

Comment:
lol!

Reply:
Nice random generator, I got some similar idea as yours. BTW, the name options lol...